internal/pkg/db/mongo/notifications.go

```diff
@@ -369,6 +369,14 @@ func (mc MongoClient) DeleteTransmission(age int64, status contract.Transmission
 	return mc.deleteAll(bson.M{"modified": bson.M{"$lt": end}, "status": status}, TRANSMISSION_COLLECTION)
 }
 
+func (mc MongoClient) GetTransmissionById(id string) (contract.Transmission, error) {
+	query, err := idToBsonM(id)
+	if err != nil {
+		return contract.Transmission{}, err
```

Comment:
I was initially unsure about the use of this type-less object (`Transmission{}`), but I see what looks (the following)  like a corresponding typecast (i.e. `transmission.ToContract()`) when the type is reified, in the 
func `getTransmissionsLimit(...){...}`:
```
	c = []contract.Transmission{}
	for _, transmission := range transmissions {
		c = append(c, transmission.ToContract())
	}
```
So I think we should be Okay 👍